vale/dictionary.jsonnet

```
@@ -233,6 +233,7 @@ local newWord(word, affixes, po) = {
     newWord('walkthrough', 'S', 'noun'),
     newWord('Webpack', '', 'noun'),
     newWord('webserver', 'S', 'noun'),
+    newWord('Wildfly', '', 'noun'),
```

It looks like it's actually WildFly, according to Wikipedia. Is that correct?

```suggestion
    newWord('WildFly', '', 'noun') { description: 'WildFly, formerly known as JBoss AS, or simply JBoss, is an application server written by JBoss, now developed by Red Hat (https://en.wikipedia.org/wiki/WildFly)', product: true },
```

---

I wonder if it's worth doing a refactor on the dictionary to add in the missing metadata (descriptions etc)...